### PR TITLE
zh: update config-overview.md for 2.0

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
update config-overview.md for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
preview: https://github.com/GMHDBJD/docs-dm/blob/config-overview-2.0/zh/config-overview.md

### CODE REVIEW

File: zh/config-overview.md [hunk 1]
```diff
@@ -9,29 +9,29 @@ category: reference
...
 
 ### 任务配置文件
 
-使用 DM-Ansible 部署 DM 集群时，`<path-to-dm-ansible>/conf` 中提供了任务配置文件模板：`task.yaml.exmaple` 文件。该文件是 DM 同步任务配置的标准文件，每一个具体的任务对应一个 `task.yaml` 文件。关于该配置文件的详细介绍，参见 [任务配置文件](task-configuration-file.md)。
+每一个数据同步任务对应一个 `task.yaml` 文件。关于该配置文件的详细介绍，参见[任务配置文件](task-configuration-file.md)。
 
 ### 创建数据同步任务
 
-你可以基于 `task.yaml.example` 文件来创建数据同步任务，具体步骤如下：
+具体步骤如下：
```
Comment (WangXiangUSTC) [inline zh/config-overview.md:24]: 是否要加上 create source 的操作？

Comment (GMHDBJD) [reply zh/config-overview.md:24]: 可不可以在[任务配置文件](task-configuration-file.md)里加呢

Comment (GMHDBJD) [reply zh/config-overview.md:24]: 可不可以直接在[任务配置文件](task-configuration-file.md)里加呢？

**Comment by GMHDBJD** [comment]
I'm sorry that your comment was deleted due to my mistake. @WangXiangUSTC

**Comment by WangXiangUSTC** [comment]
> I'm sorry that your comment was deleted due to my mistake. @WangXiangUSTC

ok, seems that you have already address my comment. Need to fix the CI. Rest LGTM

**Comment by toutdesuite** [comment]
@GMHDBJD It is recommended to merge this PR after `manage-replication-tasks.md#加载数据源配置` is merged.

File: zh/config-overview.md [hunk 2]
```diff
@@ -9,22 +9,22 @@ category: reference
...
 
-1. 复制 `task.yaml.example` 为 `your_task.yaml`。
-2. 参考[任务配置文件](task-configuration-file.md)来修改 `your_task.yaml` 文件。
+1. [使用 dmctl 将数据源配置加载到 DM 集群](manage-replication-tasks.md#加载数据源配置)
```
Comment (toutdesuite) [inline zh/config-overview.md:26]:
```suggestion
1. [使用 dmctl 将数据源配置加载到 DM 集群](manage-replication-tasks.md#加载数据源配置)。
```

**Pull request merged**